interp/declare.mli

```diff
@@ -24,7 +24,7 @@ open Decl_kinds
 
 type section_variable_entry =
   | SectionLocalDef of Evd.side_effects definition_entry
-  | SectionLocalAssum of types Univ.in_universe_context_set * polymorphic * bool (** Implicit status *)
+  | SectionLocalAssum of types Univ.in_universe_context_set * bool (* polymorphic *) * bool (* Implicit status *)
```

Comment:
Maybe use a local record here?

Reply:
Sure, can you suggest some names? [Feel free just to push to the branch if you think is quicker]

Reply:
Pushed